core/unit_test/incremental/Test11a_ParallelFor_TeamThreadRange.hpp

```diff
@@ -81,8 +81,8 @@ struct Hierarchical_ForLoop_A {
     Kokkos::fence();
     auto v_H = Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace(), v);
 
-    int check   = 0;
-    const int s = sY * sX;
+    long long int check   = 0;
```

Comment:
I am not quite sure why this started failing now, but this variable was overflowing and caused the test to fail. Similarly for the other test.

Reply:
Intel GPUs has the same issue and I modified all these variable types to int64_t for the tests to pass on them.

Reply:
We'd need to see what the int64_t alias is on that platform - maybe long or long long.

Reply:
We'd need to see what the int64_t alias resolves too on that platform - long, maybe long long. Why did we change the type from int to long long?

Reply:
Because it's large enough. :slightly_smiling_face: 
`long long` does have at least 64 bits whereas `int64_t` is optional (but unlikely to not be defined on any sensible platform).
If someone feels strongly about what to do here, let's open another pull request to discuss there. We can always resolve merge conflicts later but some change is necessary here to move forward.

Reply:
I can test long long on Intel GPUs and if it works there, I think we can use that. I don't see a reason why it should not work.

Reply:
Cool!